plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -69,6 +89,14 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
 
     // TODO move common tests from connector-specific classes here
 
+    private final ExecutorService threadPool = newCachedThreadPool(daemonThreadsNamed("base-connector-test"));
+
+    @AfterClass(alwaysRun = true)
+    public void afterClass()
+    {
+        assertThat(threadPool.shutdownNow()).isEmpty();
```

Comment:
don't do that, per comment elsewhere

Reply:
see https://github.com/trinodb/trino/pull/7306#discussion_r595939984

Reply:
see https://github.com/trinodb/trino/pull/7306#discussion_r596987518